data/en/fileopen.json

```diff
@@ -17,5 +17,13 @@
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/function/fileopen"}
 	},
 	"links": [],
-	"examples": []
+	"examples": [
+		{
+			"title":"Opens a file, reads a line then closes it.",
+			"description":"",
+			"code":"// Open File\nvar fileObject = fileOpen(\"/path/to/file.txt\");\n\n// Perform Actions\ntry\n{\n\t// Read Line\n\twriteOutput(fileReadLine(fileObject));\n}\n\n// Error Handling\ncatch(any ex)\n{\n\t// Report Exception\n\twriteDump(ex);\n}\n\n// Always Close\nfinally\n{\n\t// Close File\n\tfileClose(fileObject);\n}",
+			"result":"",
+			"runnable":true
```

Comment:
file methods are disabled on trycf so running that example will cause an exception.

Reply:
Yup - can you update the runnable flag to be false, that will remove the run button.